images/ceph/Makefile

```diff
@@ -19,8 +19,7 @@ include ../image.mk
 
 CEPH_IMAGE = $(BUILD_REGISTRY)/ceph-$(GOARCH)
 
-CEPH_VERSION = $(shell cat ceph_version)
-BASEIMAGE = $(CEPH_VERSION)-$(PLATFORM_ARCH)
+BASEIMAGE = ubuntu:18.04
```

Comment:
Why the switch to ubuntu? These were originally based on Centos before, IIRC.

Reply:
Was meaning to look at this again to build on centos instead. To pull in the mimic packages, I took the quick approach to revert to a previous version of the rook image based on ubuntu.